src/Redux/AddColleagueReducer/ColleagueReducer.test.js

```diff
@@ -0,0 +1,56 @@
+import initialState from './InitialState';
+import { SET_COLLEAGUE_DATA } from '../Types';
+import ColleagueReducer from './index';
+
+describe('Colleague Data Reducer', () => {
+  test('Setting 1 Colleague Data', () => {
+    expect(ColleagueReducer(undefined, {
+      type: SET_COLLEAGUE_DATA,
+      payload: [{
+        name: 'xyz',
+        jobTitle: 'Engineer',
+        contact: '987654321',
+      }],
+    }))
+      .toStrictEqual({
+        ...initialState,
+        data: [{
+          name: 'xyz',
+          jobTitle: 'Engineer',
+          contact: '987654321',
+        }],
+      });
+  });
+  test('Setting 2 Colleague Data', () => {
+    expect(ColleagueReducer(undefined, {
+      type: SET_COLLEAGUE_DATA,
+      payload: [{
+        name: 'xyz',
+        jobTitle: 'Engineer',
+        contact: '987654321',
+      },
+      {
+        name: 'ABC',
+        jobTitle: 'Cloud Engineer',
+        contact: '123456789',
+      }],
+    }))
+      .toStrictEqual({
+        ...initialState,
+        data: [{
+          name: 'xyz',
+          jobTitle: 'Engineer',
+          contact: '987654321',
+        },
+        {
+          name: 'ABC',
+          jobTitle: 'Cloud Engineer',
+          contact: '123456789',
+        }],
+      });
+  });
```

Comment:
remove this test case. I asked you to do it in single test case, adding one by one.